src/Bicep.Cli.IntegrationTests/ProgramTests.cs

```diff
@@ -32,7 +32,7 @@ public class ProgramTests
 
         private static Program CreateProgram(TextWriter outputWriter, TextWriter errorWriter)
         {
-            return new Program(TestResourceTypeProvider.Create(), outputWriter, errorWriter);
+            return new Program(TestResourceTypeProvider.Create(), outputWriter, errorWriter, "dev");
```

Comment:
As discussed, make "dev" into a constant in the unit test library that can be shared among these files.